Coq bench failing

Since sometime between https://ci.inria.fr/coq/job/benchmark-part-of-the-branch/563/console (29-Oct-2018 00:19) and https://ci.inria.fr/coq/job/benchmark-part-of-the-branch/564/console (29-Oct-2018 13:14) mathcomp has been failing on the Coq bench system.
~~~
=-=- Processing actions -=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
[coq-mathcomp-fingroup: make 1] Command started
+ make "-C" "mathcomp/fingroup" "-j" "1" (CWD=/home/jenkins/workspace/benchmark-part-of-the-branch/564/opam.NEW/4.05.0/build/coq-mathcomp-fingroup.dev)
- make: Entering directory '/home/jenkins/workspace/benchmark-part-of-the-branch/564/opam.NEW/4.05.0/build/coq-mathcomp-fingroup.dev/mathcomp/fingroup'
- make: Leaving directory '/home/jenkins/workspace/benchmark-part-of-the-branch/564/opam.NEW/4.05.0/build/coq-mathcomp-fingroup.dev/mathcomp/fingroup'
- make: *** No rule to make target '"Make"', needed by 'Makefile.coq'.  Stop.
[coq-mathcomp-fingroup: sh] Command started
+ sh "-c" "rm -rf '/home/jenkins/workspace/benchmark-part-of-the-branch/564/opam.NEW/4.05.0/lib/coq/user-contrib/mathcomp/fingroup'" (CWD=/home/jenkins/workspace/benchmark-part-of-the-branch/564/opam.NEW/4.05.0/build/coq-mathcomp-fingroup.dev)
~~~
Possible culprits are on the Coq side coq/coq#8812 and coq/coq#8763 (I don't see how those could lead to the error above) and (unless I messed up my dates) on the mathcomp side #219.

--- a/mathcomp/algebra/Makefile
+++ b/mathcomp/algebra/Makefile
@@ -1,10 +1,6 @@
 # -*- Makefile -*-
 
-COQPROJECT="Make"
-
-# --------------------------------------------------------------------
-include ../Makefile.common
-
-# --------------------------------------------------------------------
+COQPROJECT=Make
 COQMAKEOPTIONS=--no-print-directory
 
+include ../Makefile.common

--- a/mathcomp/all/Makefile
+++ b/mathcomp/all/Makefile
@@ -1,25 +1,7 @@
 # -*- Makefile -*-
 
-COQPROJECT="Make"
-
-# --------------------------------------------------------------------
-include Makefile.common
-
-# --------------------------------------------------------------------
+COQPROJECT=Make
 COQMAKEOPTIONS=--no-print-directory
 
 # --------------------------------------------------------------------
-.PHONY: install
-
-install:
-	$(MAKE) -f Makefile.coq install
-
-# --------------------------------------------------------------------
-.PHONY: count
-
-COQFILES := $(shell grep '.v$$' Make)
-
-count:
-	@coqwc $(COQFILES) | tail -1 | \
-	  awk '{printf ("%d (spec=%d+proof=%d)\n", $$1+$$2, $$1, $$2)}'
-
+include ../Makefile.common

--- a/mathcomp/character/Makefile
+++ b/mathcomp/character/Makefile
@@ -1,10 +1,7 @@
 # -*- Makefile -*-
 
-COQPROJECT="Make"
+COQPROJECT=Make
+COQMAKEOPTIONS=--no-print-directory
 
 # --------------------------------------------------------------------
 include ../Makefile.common
-
-# --------------------------------------------------------------------
-COQMAKEOPTIONS=--no-print-directory
-

--- a/mathcomp/field/Makefile
+++ b/mathcomp/field/Makefile
@@ -1,10 +1,7 @@
 # -*- Makefile -*-
 
-COQPROJECT="Make"
+COQPROJECT=Make
+COQMAKEOPTIONS=--no-print-directory
 
 # --------------------------------------------------------------------
 include ../Makefile.common
-
-# --------------------------------------------------------------------
-COQMAKEOPTIONS=--no-print-directory
-

--- a/mathcomp/fingroup/Makefile
+++ b/mathcomp/fingroup/Makefile
@@ -1,10 +1,7 @@
 # -*- Makefile -*-
 
-COQPROJECT="Make"
+COQPROJECT=Make
+COQMAKEOPTIONS=--no-print-directory
 
 # --------------------------------------------------------------------
 include ../Makefile.common
-
-# --------------------------------------------------------------------
-COQMAKEOPTIONS=--no-print-directory
-

--- a/mathcomp/solvable/Makefile
+++ b/mathcomp/solvable/Makefile
@@ -1,10 +1,7 @@
 # -*- Makefile -*-
 
-COQPROJECT="Make"
+COQPROJECT=Make
+COQMAKEOPTIONS=--no-print-directory
 
 # --------------------------------------------------------------------
 include ../Makefile.common
-
-# --------------------------------------------------------------------
-COQMAKEOPTIONS=--no-print-directory
-

